packages/cli/src/generator_templates/js/emails/components/Footer.jsx

```diff
@@ -12,7 +12,7 @@ export default function Footer() {
           color={grayDark}
         >
           © {new Date().getFullYear()} BookBook&nbsp;&nbsp;·&nbsp;&nbsp;
-          <a href="https://www.mailing.run" target="_blank" rel="noreferrer">
+          <a href="MM_EMAIL_PREFERENCES_URL" target="_blank" rel="noreferrer">
```

Comment:
It would be cool if in development mode, this would do something special when you click like open a page that explains how it will work for you in production